modules/core/shared/src/main/scala/scaladex/core/api/artifact/ArtifactResponse.scala

```diff
@@ -0,0 +1,3 @@
+package scaladex.core.api.artifact
+
+final case class ArtifactResponse(groupId: String, artifactId: String)
```

Comment:
It's a detail that can be changed later, but maybe we can also have the version of the artifact ?

Reply:
I would rather create another endpoint to get the versions of a particular artifact. Otherwise this endpoint would return a lot of results (we have more than 1,000,000 artifacts in the database).

Reply:
make sense!

Reply:
I can open an issue to create an endpoint that gets the versions of a given artifact 👍

Reply:
It can be interesting for a group_id/artifact_id